corehq/apps/export/models/new.py

```diff
@@ -3130,13 +3130,13 @@ def get_export_column(column):
             ),
             selected=True,
         )
-
+    unique_hash = table.name.split("_")[-1]
```

Comment:
Reminder: `table.name` here will be `ucr_{project space}_{table id}_{unique hash}`

Reply:
I assume when someone uses this file to create the table, the sheet name will get picked as the table name for the table to be created.
In that case, its useful to have the domain name in the name, So can we do it the following way
1. if `table.name` length is less than the limit, don't change it
2. else use `{project space}_{unique hash}`
3. else use `{unique_hash}` (for worst cases when even domain name is really long)

Reply:
> I assume when someone uses this file to create the table, the sheet name will get picked as the table name for the table to be created.

Only if they didn't specifiy the `Table Name` in the DET config file, then yes

> So can we do it the following way?

Sure!

Reply:
https://github.com/dimagi/commcare-hq/pull/33490/commits/82230c945d843a1c03189ac5c67f6db338ae5fc9

Reply:
> table.name here will be ucr_{project space}_{table id}_{unique hash}

nit: good to add this as a comment in code, I think its very useful.

Reply:
I was in two minds about this, so I'm going to add it!

Reply:
https://github.com/dimagi/commcare-hq/pull/33490/commits/48ca3f84e88b11636c3462b4d9fd0858dea5c3b2 cc @mkangia

Reply:
Cool, thank you